.github/scripts/rename.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# https://gist.github.com/vncsna/64825d5609c146e80de8b1fd623011ca
+set -euo pipefail
+
+# Define the input vars
+REPOSITORY=${1?Error: Please pass username/repo, e.g. prb/hardhat-template}
+REPOSITORY_OWNER=${2?Error: Please pass username, e.g. prb}
+
+echo "REPOSITORY: $REPOSITORY"
+echo "REPOSITORY_OWNER: $REPOSITORY_OWNER"
+
+# jq is like sed for JSON data
+JQ_OUTPUT=`jq \
+  --arg NAME "@$REPOSITORY" \
+  --arg AUTHOR_NAME "$REPOSITORY_OWNER" \
+  --arg URL "https://github.com/$REPOSITORY_OWNER" \
+  '.name = $NAME | .description = "" | .author |= ( .name = $AUTHOR_NAME | .url = $URL )' \
+  package.json
+`
+
+# Overwrite package.json
+echo "$JQ_OUTPUT" > package.json
+
+# Rename instances of "paulrberg/hardhat-template" to the new repo name in README.md for badges only
+sed -i -e "/Use this template/! s|paulrberg/hardhat-template|'${REPOSITORY}'|;" "README.md"
```

Comment:
This line must have either all single quotes or all double quotes otherwise would add 'owner/repo-name' instead of owner/repo-name in the README.md replacement

The correct version would be:
```sh
sed -i -e "/Use this template/! s|paulrberg/hardhat-template|"${REPOSITORY}"|;" "README.md"
```

Reply:
Sorry, I didn't know that it has to be all double quotes.

Thanks, modified now.